ref_ts/receivers.py

```diff
@@ -2,11 +2,47 @@
 from django.dispatch import receiver
 from hs_core.signals import pre_create_resource
 from ref_ts.models import RefTimeSeriesResource
+from . import ts_utils
 
 # redirect to render custom create-resource template
 @receiver(pre_create_resource, sender=RefTimeSeriesResource)
 def ref_time_series_describe_resource_trigger(sender, **kwargs):
     page_url_dict = kwargs['page_url_dict']
     url_key = kwargs['url_key']
     page_url_dict[url_key] = "pages/create-ref-time-series.html"
-
+    validate_files_dict = kwargs['validate_files']
+    metadata = kwargs['metadata']
+    try:
```

Comment:
@pkdash I think we need the try...except here as we are extracting metadata inside the try...except statement